Providers/Modules/Plugins/Common/plugin/out_oms_changetracking_file.rb

```diff
@@ -0,0 +1,530 @@
+module Fluent
+
+  class OutChangetrackingFile < BufferedOutput
+
+    Plugin.register_output('out_oms_oms.changetracking.file', self)
+	
+    # Endpoint URL ex. localhost.local/api/
+
+    def initialize
+      super
+	  
+      require 'base64'
+      require 'digest'
+      require 'json'
+      require 'net/http'
+      require 'net/https'
+      require 'openssl'
+      require 'rexml/document'
+      require 'securerandom'
+      require 'socket'
+      require 'uri'
+      require_relative 'omslog'
+      require_relative 'oms_configuration'
+      require_relative 'oms_common'
+    end
```

Comment:
Just curious - Can you "require" the existing out_oms/out_oms_blob to directly use the code from that library than copying? If the logic to talk to ODS changes later, you'll have to manage it at 2 places.